config/routes.rb

```diff
@@ -433,7 +433,7 @@
 
   mount TestUserDashboard::Engine, at: '/test_user_dashboard' if Settings.test_user_dashboard.env == 'staging'
 
-  if Settings.vsp_enironment == 'localhost' || Settings.vsp_environment == 'development'
+  if Rails.env.test? || Rails.env.development? || Settings.vsp_environment == 'development'
```

Comment:
Trevor's PR setting `Settings.vsp_enironment = 'localhost'` made it in to master so we should change this back to `Settings.vsp_enironment == 'localhost' || Settings.vsp_environment == 'development'`

Reply:
I tried removing the `Rails.env.test?` and it broke the mocked_auth routing on both my localhost testing & in Github actions.